packages/database/src/migrations/20201002053523-AddDistrictDetailsTablesLaosSchools-modifies-data.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+
+const REPORT_DISTRICT_INFO = {
+  id: 'Laos_Schools_District_Details_Table_District_Info',
+  dataBuilder: 'nonMatrixTableFromCells',
+  dataBuilderConfig: {
+    rows: ['District code', 'District population', 'Priority district'],
+    cells: [
+      [
+        {
+          key: 'District_code',
+          operator: 'ORG_UNIT_METADATA',
+          orgUnitCode: '{organisationUnitCode}',
+          field: 'code',
+        },
+      ],
+      ['SDP001'],
+      ['SPD001'],
+    ],
+    columns: ['main'],
+    entityAggregation: {
+      dataSourceEntityType: 'sub_district',
+    },
+  },
+  viewJson: {
+    name: 'District details table',
+    type: 'view',
+    viewType: 'multiValue',
+    valueType: 'text',
+  },
+};
+
+const REPORT_PROVINCE_INFO = {
+  id: 'Laos_Schools_District_Details_Table_Province_Info',
+  dataBuilder: 'nonMatrixTableFromCells',
+  dataBuilderConfig: {
+    rows: ['Province', 'Province code', 'Province population'],
+    cells: [
+      [
+        {
+          key: 'Province_name',
+          operator: 'ORG_UNIT_METADATA',
+          orgUnitCode: '{organisationUnitCode}',
+          ancestorType: 'district',
+          field: 'name',
+        },
+      ],
+      [
+        {
+          key: 'Province_code',
+          operator: 'ORG_UNIT_METADATA',
+          orgUnitCode: '{organisationUnitCode}',
+          ancestorType: 'district',
+          field: 'code',
+        },
+      ],
+      ['SDP001'],
+    ],
+    columns: ['main'],
+    entityAggregation: {
+      dataSourceEntityType: 'sub_district',
+      aggregationEntityType: 'district',
+      includeSiblingData: true,
+      aggregationType: 'SUM_PER_ORG_GROUP',
+    },
+  },
+  viewJson: {
+    name: 'Province details table',
+    type: 'view',
+    viewType: 'multiValue',
+    valueType: 'text',
+  },
+};
+
+const REPORT_NUMBER_OF_SCHOOLS_BY_TYPE = {
+  id: 'Laos_Schools_Number_Of_Schools_By_Type_Table',
```

Comment:
This doesn't have district in the id because it can be reused in the province and national level reports